src/types/message.ts

```diff
@@ -103,6 +79,28 @@ export interface StopMessage extends MessageBase {
   type: 'stop';
 }
 
+export interface GeofenceMessage extends MessageBase {
+  type: 'geofence';
+
+  /**
+    * Geofencing coordinates in the form of a rectangle.
+    */
+  geofence: {
+    'keepOut': [number, number][],
```

Comment:
lets add description for keepIn and keepOut fields too, and say they're [latitude, longitude]

lets also not surround the fields with quotes